app/serializers/search_document_serializer.rb

```diff
@@ -0,0 +1,37 @@
+class SearchDocumentSerializer < ActiveModel::Serializer
+  attributes :searchable_id, :searchable_type, :searchable, :project, :story, :url, :attachment_image
+
+  def project
+    searchable.respond_to?(:project) ? searchable.project : nil
```

Comment:
Is this using the `project_id` on the search_documents table and eager loading the project?